caas/kubernetes/provider/specs/v2_test.go

```diff
@@ -116,6 +116,7 @@ configMaps:
     foo: bar
     hello: world
 service:
+  podManagementPolicy: OrderedReady
```

Comment:
we don't want the k8s names here; they need to be generic
values should be "parallel", "serial"
key should be "podStartup"

Reply:
how about `scalePolicy`?
because it manages how to `create` and `terminate` pods.